jenkins/jobs/cd/tiup_mirror_online_ga.groovy

```diff
@@ -25,5 +24,86 @@
     properties {
         disableConcurrentBuilds{}
     }
+    parameters {
+        stringParam {
+            name('RELEASE_TAG')
+            defaultValue('nightly')
+            trim(false)
```

Comment:
Tiny: If not need `trim`, it is better not to use closure style.

Reply:
done, PTAL